.github/automation/.drone.yml

```diff
@@ -61,3 +62,29 @@ steps:
   - apt-get update && apt-get install -y git build-essential cmake
   - .github/automation/build.sh --threading omp --mode Release --source-dir $(pwd) --build-dir $(pwd)/build
   - .github/automation/test.sh --test-kind gtest --build-dir $(pwd)/build --report-dir $(pwd)/report
+---
+kind: pipeline
+name: Ubuntu18-JIT
+
+platform:
+  arch: amd64
```

Comment:
Since this is running on QEMU on x86 - should it be here, or with the other x86 Azure pipelines?
Actually, now that Azure offers Aarch64 runners (https://azure.microsoft.com/en-us/updates/azure-devops-pipelines-introduces-support-for-linuxarm64/) maybe we should move the Arm CI off of Drone CI?